eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/FastList.java

```diff
@@ -215,7 +215,10 @@ public FastList(Collection<? extends T> source)
     @Override
     public void clear()
     {
-        Arrays.fill(this.items, null);
+        for (int i = 0; i < size; i++)
+        {
+            items[i] = null;
+        }
```

Comment:
Sure, the reason I avoided it to start is that Arrays.fill we run the same code, but with a redundant bounds check. But the readability is preferred I can switch to that.